cpp/include/Ice/Communicator.h

```diff
@@ -20,41 +20,37 @@ namespace Ice
     class LocatorPrx;
     class RouterPrx;
 
-    /// The central object in Ice. One or more communicators can be instantiated for an Ice application.
-    /// @see Logger
-    /// @see ObjectAdapter
-    /// @see Properties
-    /// @see ValueFactory
+    /// Represents the main entry point into the Ice runtime. You create a communicator with
+    /// {@link Ice::initialize(int&, const char*[], InitializationData)} or another `initialize` overload.
+    /// @see ::initialize(int&, const char*[], InitializationData)
```

Comment:
Why do you qualify the `@link Ice::` and use `::initialize` for `@see`?

Reply:
Ice::initialize(correct params) doesn't auto-link for some reason, and @see requires an auto-link string.

Reply:
Updated comment.